COBOL Programming Course #1 - Learning COBOL/Front_Matter.tex

```diff
@@ -0,0 +1,19 @@
+\begin{center}
+  \vspace*{25pt}
+\includegraphics{Images/COBOL-Programming-Course.png}
+\hypertarget{cobol-programming-course-1}{%
+\section*{
+  \\[35pt]
+  \Huge COBOL Programming Course 1\[10pt]
+  \Huge Learning COBOL \\[15pt]
+  \Large Version 2.3.0}\label{cobol-programming-course-2}}
+\end{center}
+
+\pagebreak
+\hypertarget{copyright}{%
+\section*{Copyright}{
+  \normalsize COBOL Programming Course is licensed under Creative Commons 
+  Attribution 4.0 International. To view a copy of this license, visit 
+  \href{https://creativecommons.org/licenses/by/4.0}{https://creativecommons.org/licenses/by/4.0}. \\[10pt]
+  Copyright Contributors to the Open Mainframe Project's COBOL Programming Course}\label{copyright}}
+\pagebreak
```

Comment:
The Front_Matter.tex for the Learning COBOL chapter should also contain the same author section as the one on the first chapter :)

Reply:
Author section added to Front_Matter.tex of the Learning COBOL chapter.